index.html

```diff
@@ -6,20 +6,159 @@
     <title>Title</title>
 </head>
 <body>
-<header class="header">
+<header>
+
     <nav>
         <ul>
-            <li>Case Studies</li>
-            <li>Services</li>
-            <li>Contact Us</li>
+            <li><a href="#case-studies">Case Studies</a></li>
+            <li><a href="#services">Services</a></li>
+            <li><a href="#contact-us">Contact Us</a></li>
         </ul>
     </nav>
+
     <div class="introtext">
         <h1>Your website is an investment in the success of your business.</h1>
-        <h4>We are here to help you succeed.</h4>
-        <button type="button">Let's work together</button>
+        <h3 class="introtexth3">We are here to help you succeed.</h3>
+        <button type="button" class="headerbutton"><a href="contact-us">Let's work together</a></button>
     </div>
-    <img>
+
+    <img class="squiggle" src="../assets/squiggle_line_stripe.svg" alt="squiggle-decoration">
```

Comment:
Voor alle plaatjes geldt dat de `../` weg moet
```suggestion
    <img class="squiggle" src="assets/squiggle_line_stripe.svg" alt="squiggle-decoration">
```

De assets folder staat immers naast de index.html. Nu kan ik bij het openen hiervan in de browser de plaatjes niet zien